cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js

```diff
@@ -82,18 +85,20 @@ function FileActionsDropdown({ file, table_type, setOpenModal, setSelectedFile,
           title="Download this file in JSON format"
           download
         >
-          Download json of all files
-        </a>
-        <a
-          className="item ui fluid button"
-          href={`${getFileUri(table_type, file.key)}?qos=hot`}
-          target="_blank"
-          rel="noopener noreferrer"
-          title="Download this file in JSON format"
-          download
-        >
-          Download json of hot files
+          Format: json
         </a>
+        { !isOnline && (
+            <a
+              className="item ui fluid button"
+              href={`${getFileUri(table_type, file.key)}?qos=hot`}
```

Comment:
nit: I would try to pass down the values from the `ColdStorageActions` so we don't hardcode the values `hot`, `cold` etc.

Reply:
Good point. I'll create a ticket to change that (since that requires also a couple of changes in the python class and I don't dare to do it right now)